README.md

```diff
@@ -1,7 +1,14 @@
 # FFC Pay Request Editor
 
-## Prerequisites
+## Description
+This is a microservice, web front-end which performs debt enrichment and ledger assignment overrides. 
```

Comment:
FFC web front-end microservice which... (to be consistent with your other READMEs)